src/modules/claims/claims.service.ts

```diff
@@ -441,6 +452,19 @@ export class ClaimsService {
     await this._cacheClient.issueClaim(this._signerService.did, message);
   }
 
+  async verifyVc(vc: VerifiableCredential<RoleCredentialSubject>) {
```

Comment:
Missing documentation

Reply:
@JGiter I added documentation for this, let me know if it is not what you would have put

Reply:
We can also stress out that this method verifies offchain credential
``` Verifies offchain role credential according with role definition```